### PR TITLE
chore: Release a Docker image

### PR DESCRIPTION
**Pull request by danielpacak**
Resolves: #10

Before merging we have to encrypt and add some envs to .travis.yml

GoReleaser requires the following environment variables to be set:

| Environment Variable | Description |
|----------------------|-------------|
| `GITHUB_TOKEN`       | GitHub API token with the `repo` scope to deploy the artifacts to GitHub |
| `DOCKERHUB_USER`     | DockerHub username |
| `DOCKERHUB_TOKEN`    | DockerHub access token to push images |

```
$ travis encrypt GITHUB_TOKEN="***" --repo aquasecurity/harbor-scanner-trivy
$ travis encrypt DOCKERHUB_USER="***" --repo aquasecurity/harbor-scanner-trivy
$ travis encrypt DOCKERHUB_TOKEN="***" --repo aquasecurity/harbor-scanner-trivy
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+language: go
+go:
+  - "1.12"
```
Comment (knqyf263) [inline .travis.yml]: 1.13 is ready.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+git:
+  depth: 1
+
+services:
+  - docker
+
+script:
+  - make test
+  - curl -sL https://git.io/goreleaser | bash -s -- --snapshot --skip-publish --rm-dist
+  - docker image ls
```
Comment (knqyf263) [inline .travis.yml]: Is it necessary?

File: .travis.yml (hunk 1 above)
Comment (danielpacak) [reply .travis.yml]: I've added TODO to handle migration in a dedicated PR. I have this GO111MODULE=on env spread in few places and I need more testing to make sure that upgrade does not break things.

File: .travis.yml (hunk 2 above)
Comment (danielpacak) [reply .travis.yml]: Nope. It was there just for debugging. I removed it.

**Pull request merged**